blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
@afercia 
My opinion is that if JAWS (the screen reader which used to be the only one in town) supports aria-label, we should go ahead and use that instead of the visually hidden ones.
Is there any way we can find out?